OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs

```diff
@@ -59,6 +59,9 @@ public Player Owner
 
 			set
 			{
+				if (value == null || !(value is Player))
```

Comment:
This is a wild guess to be honest. Even the null check. You suggested on IRC to add this. I can't really prove this fixes anything and I believe the Lua bridge will crash on a lower level part if you throw wrong parameters on it "can't convert stupid scripter typo without naming even the variable nil to LuaValue in line number 30 in sandbox.lua". I also tried to improve the stacktrace a bit in those situations as it is _really_ annyoing, but failed. Will remove this and hope for @Unit158 to finish #9086.